phalcon/cache/backend/redis.zep

```diff
@@ -88,13 +99,21 @@ class Redis extends Backend implements BackendInterface
 		}
 
 		if !isset options["statsKey"] {
-			// Disable tracking of cached keys per default
-			let options["statsKey"] = "";
+			let options["statsKey"] = "_PHCR";
```

Comment:
> Modify cache redis backend, delete _PHCR

As I remember it was done in  https://github.com/pantaovay/cphalcon/commit/6b8842267d875e5d2a1a58b701e758b909e53dc9

Why are we again added _PHCR ?